webclient/src/views/search/view-search.js

```diff
@@ -39,7 +39,11 @@ navigatum.registerView('search', {
         loadSearchData: function(query, data) {
             this.search_data = data;
             this.query = query;
-            
+            var search= document.getElementById("search")
+            if (search.value.length === 0) {
+                // /search/:querry is called from an internal url and thus the search url is not set
+                search.value = query;
+            }
```

Comment:
What do you think about this? 
is this nessesary?
Through the bug in the ngnix config I am unable to check, if this could work

Reply:
It seems not. When I use hash based navigation, vue-router shows a 404 for `search/` without a query.
Instead of using the `document.getElementById` approach you can however set the query using vue.js directly. Changing the element works as well, but this would be more consistent with the rest of the app.

```suggestion
            navigatum.app.search.query = query;
```

Reply:
Thanks :)